scripts/exploration.gd

```diff
@@ -574,52 +574,97 @@ func _on_confirmwinning_pressed(): #0 leave, 1 capture, 2 rape, 3 kill
 				else:
 					text += baddie.quirk("\n[color=yellow]-"+str(globals.randomitemfromarray(['No! Please stop this!','I am going to be sick!','N-n-no!',"Please don't do this!",'Why me?!','Please stop!','Someone help me!',"I'll do anything! Please, no!"]))+ "[/color]")
 				relations = round(rand_range(-10,-20)) - (baddie.npcexpanded.timesraped*5)
 				baddie.fear = round(baddie.npcexpanded.timesraped*2.5)+1
-			text += "\n"
+				text += '\n'
+
+			if friend != null:  ### we react to the captive. this could be relatively short. could make a number of actions based on flaws, too...
+				if (friend.checkVice('wrath') || friend.checkFetish('sadism')) && mean == true:
+					text += friend.dictionary(globals.randomitemfromarray(['$name angrily grabs ', '$name growls and clutches ']))
+					text += baddie.dictionary(globals.randomitemfromarray(['$his hair as $he protests, and strikes $his face again and again until $he is reduced to mere whimpers. ','$his throat until $he can no longer protest. ']))
+					friend.stress -= rand_range(10,20)
+				elif friend.checkFetish('submission') && mean == false:
+					text += friend.dictionary(globals.randomitemfromarray(['$name kneels next to ','$name smiles to ']))
+					text += baddie.dictionary(globals.randomitemfromarray(['$him and whispers encouragements into $his ear. ', '$him and strokes $his hair as $he coooperates. ']))
+				elif friend.checkFetish('bondage') && mean == true:
+					text += friend.dictionary(globals.randomitemfromarray(['$name tightens some rope around ','$name collects some nearby vines, then binds ']))
+					text += baddie.dictionary(globals.randomitemfromarray(['$his arms and chest and $his struggles become more labored. ','$him and lifts $him into position for you. ']))
+				else:
+					text += friend.dictionary(globals.randomitemfromarray(['$name gives you a knowing smile ','$name grins in anticipation ']))
+					text += baddie.dictionary(globals.randomitemfromarray(['while $he squirms. ','as you approach. ']))
+				text += '\n'
+
 			if globals.player.penis != "none":
 				if baddie.vagina != "none":
 					if baddie.vagvirgin == true:
 						baddie.vagvirgin = false
 						text += baddie.dictionary("\nYou slam your "+str(globals.expansion.namePenis())+" into $his tight, virgin "+str(globals.expansion.namePussy())+" mercilessly. You hear $him sob as $his hymen rips but keep fucking $him ")
 					else:
 						text += baddie.dictionary("\nYou slam your "+str(globals.expansion.namePenis())+" into $his "+str(globals.expansion.namePussy())+" relentlessly. You occassionally switch to pounding $his "+str(globals.expansion.nameAsshole())+" ")
-					if baddie.assvirgin == true:
-						baddie.assvirgin = false
-						baddie.cum.ass += round(globals.player.pregexp.cumprod*.4)
+						if baddie.assvirgin == true:
```

Comment:
Is there a reason why the player will only rape the baddie's ass if they are not a virgin? There is no text for ass raping. Also it appears that the player is only straight and will not rape the ass of a male if the player is gay.